performance/kops_cluster_setup/create_agentscale_yaml.sh

```diff
@@ -6,8 +6,8 @@
 # Define variables
 export AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id)
 export AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key)
-BUCKET=agentscale-dev-scalyr-com-state-store
-export KOPS_CLUSTER_NAME=agentscale.dev.scalyr.com
+BUCKET=<YOUR_BUCKET_STORE_NAME>
+export KOPS_CLUSTER_NAME=<YOUR_CLUSTER_NAME>  # should be a valid DNS name, e.g. <YOUR_CUSTER_NAME>
```

Comment:
> should be a valid DNS name, e.g. <YOUR_CUSTER_NAME>

`<YOUR_CLUSTER_NAME>` is not a valid DNS name :D

Reply:
Whoops.  Global search and replace mistake.  I've restored it to `agentscale.dev.scalyr.com`